components/lib/inputnumber/InputNumber.js

```diff
@@ -75,11 +76,16 @@ export const InputNumber = React.memo(
             _minusSign.current = getMinusSignExpression();
             _currency.current = getCurrencyExpression();
             _decimal.current = getDecimalExpression();
+            _decimalSeparator.current = getDecimalSeparator();
```

Comment:
I don't understand the difference between `decimalSeparater.current` and `decimal.current` I think `decimal.current` was supposed to be the current decimal place regardless of whether its a `.` or a `,` ?

Reply:
`decimal.current` is a regular
 `/[,]/g`, `/[.]/g`, `/[]/g`
```
const getDecimalExpression = () => {
            const formatter = new Intl.NumberFormat(_locale, { ...getOptions(), useGrouping: false });

            return new RegExp(`[${formatter.format(1.1).replace(_currency.current, '').trim().replace(_numeral.current, '')}]`, 'g');
        };
```